_pages/en/web-services-access.md

```diff
@@ -22,22 +22,22 @@ The policy applies only to unclassified interactions (data, information, code, e
 ### Issue
 
 ESDC is currently blocking access to some popular and useful collaboration Web tools like Google Docs, Google Hangouts and OneDrive.
-Even though TBS policy recommends open access to these tools by default, ESDC's rationale for blocking them is primarily to prevent accidental leak of protected information by employees.
+Even though TBS policy recommends open access to these tools by default, ESDC's rationale for blocking them is primarily to prevent an accidental leak of protected information by employees.
 
-The already proposed risk mitigation is to augment the Data Loss Prevention (DLP) capability with a tool that enforces classification of documents and emails.
-However, as the evaluated options so far can be costly, implementation is not being considered for the short term.
+A proposed mitigation item to reduce this risk is to augment the Data Loss Prevention (DLP) capability with a tool that enforces classification of documents and emails.
+However, as the evaluated options to date are costly, implementation is not being considered for the short term.
```

Comment:
Should we comment who is making that decision?

Reply:
Not sure it's needed. Regardless of who, we still need this capability in place.

Reply:
Agreed with @gabrielcossette